extensions/debezium-mongodb/runtime/pom.xml

```diff
@@ -31,7 +31,7 @@
 
     <properties>
         <camel.quarkus.jvmSince>1.0.0</camel.quarkus.jvmSince>
-        <camel.quarkus.nativeSince>1.0.0</camel.quarkus.nativeSince>
+        <camel.quarkus.nativeSince>1.5.0</camel.quarkus.nativeSince>
```

Comment:
From the time of the original PR, version should be bumped to 1.6.

Reply:
Good catch, thanks!